config/application.yml.default

```diff
@@ -526,6 +526,7 @@ test:
   hmac_fingerprinter_key: a2c813d4dca919340866ba58063e4072adc459b767a74cf2666d5c1eef3861db26708e7437abde1755eb24f4034386b0fea1850a1cb7e56bff8fae3cc6ade96c
   hmac_fingerprinter_key_queue: '["old-key-one", "old-key-two"]'
   identity_pki_disabled: true
+  in_person_state_id_controller_enabled: true
   lexisnexis_trueid_account_id: 'test_account'
```

Comment:
Good read:-  Lifecycle of a feature flag](https://handbook.login.gov/articles/appdev-feature-flags.html) - I think we should have set default but this has been in development so long that I don't think that would have been a good idea so point this out for future Joy. (I know you were not here when this was started.)